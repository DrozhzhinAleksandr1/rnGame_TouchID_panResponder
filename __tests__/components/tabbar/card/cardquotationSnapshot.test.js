import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { CardQuotation } from '../../../../app/components/tabbar/card/cardquotation';

const mockItem = { Bid: "0.95068", titleOfSell:'Продажа' };

it('create snapshot Card', () => {
    const renderer = new ShallowRenderer()
    let cardScreenSnapshot = renderer.render(<CardQuotation numbFromServer={mockItem.Bid} title={mockItem.titleOfSell} />);
    expect(cardScreenSnapshot).toMatchSnapshot();
});