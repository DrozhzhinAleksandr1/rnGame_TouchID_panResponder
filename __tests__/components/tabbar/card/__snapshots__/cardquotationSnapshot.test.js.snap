// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot Card 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "height": "100%",
    }
  }
>
  <Component>
    <Component>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#5A6772",
            "fontFamily": "Roboto-Light",
            "fontSize": 11,
            "marginBottom": 5,
          }
        }
      >
        Продажа
      </Text>
    </Component>
    <Component
      style={
        Object {
          "flex": 1,
          "flexDirection": "row",
          "marginBottom": "5%",
          "marginTop": "2%",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#fff",
            "fontFamily": "Roboto-Light",
            "fontSize": 20,
          }
        }
      >
        0.95068
      </Text>
    </Component>
  </Component>
</Component>
`;
